composer.json

```diff
@@ -22,7 +22,9 @@
     "require": {
         "php": "^8.1",
         "ext-intl": "*",
-        "nesbot/carbon": "^2.72.1|^3.0"
+        "nesbot/carbon": "^2.72.1|^3.0",
+        "ext-calendar": "*",
+        "solaris/php-moon-phase": "^2.1"
```

Comment:
solaris/php-moon-phase can be removed, I think?

Reply:
I think I used the moon phase for deepavali event as this holiday is based on the moon phases